foundation/src/main/java/com/decathlon/vitamin/compose/foundation/Shape.kt

```diff
@@ -5,7 +5,7 @@ import androidx.compose.material.Shapes
 import androidx.compose.ui.unit.dp
 
 val shapes = Shapes(
-    small = RoundedCornerShape(2.dp),
-    medium = RoundedCornerShape(2.dp),
-    large = RoundedCornerShape(2.dp)
+    small = RoundedCornerShape(vtmnRadius.radius100),
+    medium = RoundedCornerShape(vtmnRadius.radius200),
+    large = RoundedCornerShape(0.dp)
```

Comment:
On Figma, the `large` version has rounded borders.
You might have missed something here with `0dp`.

Reply:
According to the Shapes.large documentation, the `large` version is used by large components like `ModalDrawer` or `ModalBottomSheetLayout`. These components do not have rounded corners in Vitamin.
Please share me the documentation link if you see otherwise.

Reply:
The M2 value for large shape is `0.dp`. Because we don't have an official mapping between M2 and Vitamin, we can/should keep this value for now and come back later when we'll have an official mapping.